tree/ntuple/test/StreamerField.hxx

```diff
@@ -67,20 +67,26 @@ class IgnoreUnsplitComment {
 // Test streamer field with polymorphic type
 
 struct PolyBase {
-   virtual ~PolyBase() {}
+   virtual ~PolyBase() = default;
    int x;
+   ClassDef(PolyBase, 3);
```

Comment:
I find surprising that this help and is likely indicative of some other (now obscured) problem (i.e. if this change actually makes a difference). ... 

One possibility is the anchoring (by the compiler) of the virtual function table (usually associated with the first virtual function).

However the issue seems to be reported from `CloseStreamerInfoROOTFile` which should not be related to the virtual function table ...

Reply:
Opened https://github.com/root-project/root/pull/20373 https://github.com/root-project/root/pull/20374 to see which one of the two changes is necessary, if not both.